src/blurredLocation.js

```diff
@@ -245,12 +261,35 @@ BlurredLocation = function BlurredLocation(options) {
       options.map.on('move', addScaleToListener);      
     }
     else {
-      $("#"+ options.scaleDisplay).text("");
+      $("#" + options.scaleDisplay).text("");
       options.map.off('move', addScaleToListener);
     }
   }
 
+  function getBlurryScale() {
+    var urban = options.blurryScaleDict["urban_"+getPrecision().toString()]
```

Comment:
Cool! Maybe it should be `getBlurryScale(region)` where `region = region || 'rural'` for a default?

Reply:
Done!!